PHP/tests/syntax_test_php.php

```diff
@@ -1935,12 +1935,12 @@ function generate2()
 //       ^^^ entity.name.tag.heredoc
 one: two
 //^^^^^^ meta.embedded.yaml source.yaml
-//^ string.unquoted.plain.out entity.name.tag
+//^ meta.mapping.key string
```

Comment:
Was there a reason why the sub-scope(s) of `string` are no longer asserted here? 
```suggestion
//^ meta.mapping.key string.unquoted
```

Reply:
I made the test more general so that it wouldn't break as easily when YAML is updated again. I don't think we'd remove the `unquoted` part, but the purpose of the test is primarily to check whether a freuqent use case of embedding works, not all the details of the YAML syntax. `meta.mapping.key string` seems to me to be the most general assertion that would apply here, especially seeing that it's likely to be used by color schemes as well.

Reply:
Makes sense, thanks for the explanation.